enterprise/server/remote_execution/platform/platform_test.go

```diff
@@ -189,31 +190,31 @@ func TestParse_ApplyOverrides(t *testing.T) {
 		platformProps       []*repb.Platform_Property
 		startingEnvVars     []*repb.Command_EnvironmentVariable
 		expectedEnvVars     []*repb.Command_EnvironmentVariable
+		errorExpected       bool
```

Comment:
don't think it's necessary here, but if you'd like to check for specific errors, you could have this be an *error and compare it for equality if set.

Reply:
I had that to start. It felt a little heavy to have to creat an error in the setup, since we control the stubbed function as well.